pandas/io/common.py

```diff
@@ -276,56 +276,92 @@ def ZipFile(*args, **kwargs):
     ZipFile = zipfile.ZipFile
 
 
-def _get_handle(path, mode, encoding=None, compression=None):
-    """Gets file handle for given path and mode.
+def _get_handle(source, mode, encoding=None, compression=None):
     """
-    if compression is not None:
-        if encoding is not None and not compat.PY3:
+    Gets file handle for given source and mode.
+    wraps compressed fileobject in a decompressing fileobject
+    NOTE: For all files in Python 3.2 and for bzip'd files under all Python
```

Comment:
we want to remove this statement and its functionaility as we don't support 3.2 anymore